python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -0,0 +1,602 @@
+"""
+Linear Regression with interval-censored observations
+=====================================================
+"""
+
+# %%
+# 1. Model formulation
+# --------------------
+#
+# We consider the following linear model:
+#
+# .. math::
+#
+#    \vect{Y} = \mat{X} \vect{\theta} + \vect{\varepsilon},
+#
+# where the observation vector :math:`Y \in \mathbb R^{n}` is modeled as
```

Comment:
Please check the example thoroughly for typos. All vectors and matrices should be bold (which is done with the \vect and \mat macros).